easybuild/easyconfigs/b/beagle-lib/beagle-lib-4.0.0-GCC-11.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'beagle-lib'
+version = '4.0.0'
+
+homepage = 'https://github.com/beagle-dev/beagle-lib'
+description = """beagle-lib is a high-performance library that can perform the core calculations at the heart of most
+ Bayesian and Maximum Likelihood phylogenetics packages."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+github_account = 'beagle-dev'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['d197eeb7fe5879dfbae789c459bcc901cb04d52c9cf5ef14fb07ff7a6b74560b']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+]
+
+dependencies = [
+    ('Java', '11', '', SYSTEM),
+    ('pkgconf', '1.8.0'),
```

Comment:
is `pkgconf` really a runtime dependency?

Reply:
good question, probably not, but this was just copied over from older easyconfigs so i'm not sure